src/controllers/expensesController.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const userService = require('../services/users.js');
+const expensesService = require('../services/expenses.js');
+
+const getAll = (req, res) => {
+  const expenses = expensesService.getAll(req.query);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const expenseId = Number(req.params.expenseId);
+
+  if (isNaN(expenseId)) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundedExpense = expensesService.getById(expenseId);
+
+  if (!foundedExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundedExpense);
+};
+
+const createExpense = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = userService.getUser(userId);
+
+  if (!foundUser) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = expensesService
+    .create(userId, spentAt, title, amount, category, note);
+
+  res.statusCode = 201;
```

Comment:
you can use method for it
```suggestion
  res.status(201);
```